README.rst

```diff
@@ -350,8 +350,7 @@ See also the `MAGICC website <http://magicc.org/>`_ and
 `Wiki <http://wiki.magicc.org/index.php?title=Main_Page>`_
 for further information.
 
-The ``pymagicc`` wrapper is free software under the GNU Affero General Public
-License v3, see `LICENSE <./LICENSE>`_.
+The ``pymagicc`` wrapper is free software. For license details, see `LICENSE <./LICENSE>`_.
```

Comment:
```suggestion
The ``pymagicc`` wrapper itself is licensed under a BSD-3 license. For details, see `LICENSE <./LICENSE>`_.
```
"Free" software is historically understood to be different from "Open Source" software.
https://en.wikipedia.org/wiki/Free_and_open-source_software

Reply:
Ah ok. Shall we add free too then?

Reply:
```suggestion
The ``pymagicc`` wrapper itself is free software released under a BSD-3 license. For license details, see `LICENSE <./LICENSE>`_.
```

Reply:
Technically, BSD is free software but i think it's confusing as i myself was confused and thought "free" only applied to GPL-style software. I think people even further away from these FOSS dialectics (like potential scientist users) and the "free as in speech or free as in beer" differences i'd still leave "free" out probably. (Or write "open source")

Reply:
ok easy